assignment1.txt

```diff
@@ -0,0 +1,23 @@
+Matthew Kellerman
+1.
```

Comment:
right, more specifically, a compiler takes your code written in C (or some other high level language), removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.